tests/queries/0_stateless/03280_aliases_override_logic.sql

```diff
@@ -0,0 +1,27 @@
+SELECT b FROM
+(
+    SELECT number, number*2
+    FROM numbers(2)
+) as x (a, b);
+
+SELECT a FROM
+(
+    SELECT number, number*2
+    FROM numbers(2)
+) as x (a, b);
+
+SELECT a FROM
+(
+SELECT number, number*2
+FROM numbers(10)
+) as x (a); -- { serverError BAD_ARGUMENTS }
+
+CREATE VIEW IF NOT EXISTS test_view_03280 (a,b) AS SELECT 1, 2;
+
+SELECT a FROM test_view_03280;
+
+SELECT b FROM test_view_03280;
+
+DROP VIEW IF EXISTS test_view_03280;
+
+CREATE VIEW test_view_1_03280 (a) AS SELECT 1, 2; -- { serverError BAD_ARGUMENTS }
```

Comment:
Consider adding some degenerate cases like

`CREATE VIEW test_view_1_03280 (a, b] AS SELECT 1, 2;`

and

`CREATE VIEW test_view_1_03280 ((a, b)) AS SELECT 1, 2;`